maddymz/irondb
# US432

Refactored program to graph analysis techniques by count and page number.

To Test:
1. If you open the file in an editor, disable pylint or other linting.
2. cd irondb/external_modules/pdfScraper
3. source activate journalImport
4. python pdf_analysis_driver.py
5. Wait for program to finish running and a graph should be displayed.

Expected output is:

![image](https://user-images.githubusercontent.com/24575856/51222307-01088a80-18f2-11e9-9948-70db68bb070c.png)


If errors:
You may need to "conda install matplotlib" as it wasn't in the environment.
